docs/guide/walletconnect.md

```diff
@@ -0,0 +1,12 @@
+# WalletConnect
+
+You can use MetaMask mobile as a signer while using applications on another browser, desktop, or mobile application with WalletConnect.
+
+For a dapp developer, they wouldn’t need to do anything other than use our “official” qrcode-modal which already includes MetaMask deep linking
```

Comment:
For a dapp developer, they wouldn’t need to do anything other than use **their** “official” qrcode-modal which already includes MetaMask deep linking